daal4py/sklearn/monkeypatch/dispatcher.py

```diff
@@ -76,6 +81,8 @@
     'lasso':     [[(linear_model_module, 'Lasso', Lasso_daal4py), None]],
     'svm':       [[(svm_module, 'SVC', SVC_daal4py), None]],
     'logistic':  [[(logistic_module, _patched_log_reg_path_func_name, daal_optimized_logistic_path), None]],
+    'knn_clsf':  [[(neighbors_module, 'KNeighborsClassifier', KNeighborsClassifierOneDAL), None]],
+    'knn_mxn':   [[(neighbors_module, 'KNeighborsMixin', KNeighborsMixinOneDAL), None]],
```

Comment:
Can make it consistency with other wrapper for now? And then rename everything together. It looks very strange :)
```suggestion
    'knn_mxn':   [[(neighbors_module, 'KNeighborsMixin', KNeighborsClassifier_daal4py), None]],
```

Reply:
Fixed

Reply:
Renamed